lib/views/after_auth_screens/events/event_info_page.dart

```diff
@@ -52,7 +52,7 @@ class _EventInfoPageState extends State<EventInfoPage> {
                   label: Text(
                     model.fabTitle,
                     style: Theme.of(context).textTheme.bodyText2!.copyWith(
-                          color: Theme.of(context).colorScheme.secondary,
+                          color: Theme.of(context).colorScheme.onBackground,
```

Comment:
Can you write tests for [lib/views/after_auth_screens/events/event_info_page.dart](https://github.com/PalisadoesFoundation/talawa/pull/1265/files#diff-d4e6f66e4d16a0e65c8c3da52c6ed1ebf78271b4b41a8599ad2107012d2dedad)? We really need to get the code more reliable. If we had written a test for this, you wouldn't have seen the bug.

Reply:
Yes, I'll write the test cases for the required file.